deployment/centos/deploy.sh

```diff
@@ -1,27 +1,41 @@
 #!/bin/bash -e
 
-# change this to the current semver tag, e.g. v1.8.0
-GIT_TAG=master
+# deploy scoreboard to a centos environment
+
+# usage: 
+#   deploy.sh {tag or branch}
+#
+# example:
+#   deploy.sh v1.8.0
+#
+
+GIT_TAG=${1:-master}
+NGINX_PATH=/var/www
 
 echo "using git tag or branch: ${GIT_TAG}"
 echo
 
 cd ~
+
+# find the tag or branch
 rm -rf scoreboard
 git clone https://github.com/developmentseed/scoreboard.git
-cd scoreboard
+pushd scoreboard
 git checkout ${GIT_TAG}
 
-# installation
-npm install
-npm run migrate
-npm run build
+# install & build steps
+yarn install
+yarn run migrate
+yarn run build
+popd
 
-# move to nginx folder
-cd ../..
-suffix=$(date +%s)
-sudo mv /var/www/scoreboard /var/www/scoreboard_$suffix || echo 'this is the first deployment'
-sudo mv scoreboard /var/www/
+# update nginx directory
+datestamp=$(date +%F)
+if [ -d "${NGINX_PATH}/scoreboard" ]; then
+    # backup previous deployment
+    sudo mv -v ${NGINX_PATH}/scoreboard ${NGINX_PATH}/scoreboard_${datestamp}
+fi
+sudo mv -v scoreboard ${NGINX_PATH}
 
 # restart services
 sudo systemctl restart scoreboard-api
```

Comment:
Maybe we restart scoreboard-timer.service for good measure

Reply:
@kamicut good point. I will look into this- either it is required or not, not for good measure :)
I think that `scoreboard-timer.service` is not running, rather the `scoreboard-timer.timer` starts it periodically but will double-check this.

Reply:
@kamicut actually I believe `sudo systemctl restart scoreboard-timer.service` and `sudo systemctl restart scoreboard-timer` are identical commands. I updated the syntax though, see my last commit.